### PR TITLE
Homepage QA fixes

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Reduces whitespace on the desktop and mobile header
Screen Shot 2020-02-05 at 1 37 18 PM

- Reduces height of featured image
Screen Shot 2020-02-05 at 1 37 42 PM

- Centers member station descriptive text
- Adds map legend
- Reduces top and bottom spacing on map
Screen Shot 2020-02-05 at 1 38 53 PM

- Decreases spacing on member stations list
Screen Shot 2020-02-05 at 1 39 25 PM

- Adds spacing above/below video widget
Screen Shot 2020-02-05 at 1 39 57 PM

- Adds "More Events" button styling
Screen Shot 2020-02-05 at 1 40 23 PM

- Widens sponsors slider
Screen Shot 2020-02-05 at 1 40 50 PM

- Reduces size of footer member station links
Screen Shot 2020-02-05 at 1 41 13 PM

## Why

For #35

## Testing/Questions

Features that this PR affects:

- Site header
- Homepage featured area background image
- Member map widget
- Member stations widget
- Video embed widget area
- Sponsors slider
- Footer area lists
- Buttons

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?

Steps to test this PR:

**Desktop and mobile header**
1. View the frontend and verify the new reduced spacing looks ok

**Homepage featured area background image**
1. View the frontend and verify the reduced background image size still looks ok

**Member map**
1. View the widget and verify the legend displays now
2. Make sure the new map size looks ok

**More Events button for calendar**
1. Add a custom HTML widget under the calendar widget and paste this
```
<div style="text-align:center;">
	<a href="#" class="btn btn-blue">More Events</a>
</div>
```
2. View on the frontend and verify it looks ok

**Video embed**
1. Make sure new spacing looks ok

**Sponsors slider**
1. View the slider and make sure the slider is wider than it previously was

**Footer member station links**
1. View the footer menu station links and make sure the new font size looks ok

### CODE REVIEW

File: wp-content/themes/ipbs-largo/less/mixins.less [hunk 1]
```diff
@@ -19,9 +19,12 @@
     color: @white;
     font-weight: 600;
     border-radius: 20px!important;
-    padding: 10px 40px!important;
+    padding: 10px 50px!important;
     text-transform: uppercase;
     text-decoration: none;
+    box-shadow: 0px 0px 24px @box-shadow-black;
```
Comment (benlk) [inline wp-content/themes/ipbs-largo/less/mixins.less]: Screen Shot 2020-02-05 at 16 39 11

The `.btn` styles apply to the button in the mobile nav. 😢

Comment (joshdarby) [reply wp-content/themes/ipbs-largo/less/mixins.less]: Should be fixed in https://github.com/INN/umbrella-ipbs/pull/83/commits/974bdf19a16285bf322d89cab2759077cd538b5f. Good catch 😅

File: wp-content/themes/ipbs-largo/less/map.less [hunk 2]
```diff
@@ -1,55 +1,24 @@
...
+.ipbs-member-map-widget {
+  margin-bottom: 0!important;
   #legend {
     font-size: 16px;
     line-height: 16px;
     font-family: @font-map-labels;
     color: #221F1F;
     border: 2px solid #d1d2d3;
     padding: 0.5em;
+    width: fit-content;
```
Comment (benlk) [inline wp-content/themes/ipbs-largo/less/map.less]: `fit-content` isn't widely supported at this time: https://caniuse.com/#search=fit-content

`max-content` works here, I think?

File: wp-content/themes/ipbs-largo/less/map.less [hunk 3]
```diff
@@ -1,55 +1,24 @@
...
-
+.ipbs-member-map-widget {
+  margin-bottom: 0!important;
   #legend {
     font-size: 16px;
     line-height: 16px;
     font-family: @font-map-labels;
     color: #221F1F;
     border: 2px solid #d1d2d3;
     padding: 0.5em;
```
Comment (benlk) [inline wp-content/themes/ipbs-largo/less/map.less:11]: this padding is superfluous as it's overwritten later in this same style block.

**Review by benlk — CHANGES_REQUESTED** [review]
The only big blocker now that button styles are fixed is `fit-content`.

File: wp-content/themes/ipbs-largo/less/map.less (hunk 3 above)
Comment (joshdarby) [reply wp-content/themes/ipbs-largo/less/map.less:11]: Removed in https://github.com/INN/umbrella-ipbs/pull/83/commits/538da70a6325e3abed525fc039e22d100dfa9df2

File: wp-content/themes/ipbs-largo/less/map.less (hunk 2 above)
Comment (joshdarby) [reply wp-content/themes/ipbs-largo/less/map.less]: Updated in https://github.com/INN/umbrella-ipbs/pull/83/commits/538da70a6325e3abed525fc039e22d100dfa9df2

**Review by benlk — APPROVED** [review]

**Pull request merged**